src/SFA.DAS.AANHub.Api.UnitTests/Controllers/EmployersControllerTests.cs

```diff
@@ -46,21 +52,23 @@ public EmployersControllerTests()
             result.As<CreatedAtActionResult>().StatusCode.Should().Be(StatusCodes.Status201Created);
```

Comment:
Add assertion on the Id in the response if possible. But I guess this needs changing anyway to take two parameters

Reply:
Done